nodejs/awsx/ecs/taskDefinition.ts

```diff
@@ -155,7 +155,10 @@ export abstract class TaskDefinition extends pulumi.ComponentResource {
 
     // Default policy arns for the Execution role.
     public static defaultExecutionRolePolicyARNs() {
-        return ["arn:aws:iam::aws:policy/service-role/AmazonECSTaskExecutionRolePolicy"];
+        return [
+            "arn:aws:iam::aws:policy/service-role/AmazonECSTaskExecutionRolePolicy",
+            aws.iam.ManagedPolicies.CloudWatchLogsFullAccess,
```

Comment:
Without this policy, there is no way for our service to be able to create it's LogGroup

Reply:
The `AmazonECSTaskExecutionRolePolicy` policy provides the needed CloudWatch actions.  This change should not be necessary.  What exactly did you hit that made you think it was necessary?

https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_execution_IAM_role.html

Reply:
It allows interacting with cloudwatch but it doesnlt allow creating a log group - that's handled by logs.CreateLogGroup

Reply:
Why does it need to create a log group? What issue were you hitting?

Reply:
It currently creates a log group. At the moment the end user has no control over the name and details (which is the issue that the user is having) and so they can't create one for the firelens logs

Reply:
@lukehoban any extra thoughts here?

Reply:
I think one thing we could do here if we want to keep the permissions scoped to a minimum is actually attach `AWSLambdaBasicExecutionRole` instead, which provides just:

```
"logs:CreateLogGroup",
"logs:CreateLogStream",
"logs:PutLogEvents"
```

which I think is all that's needed vs. giving full access to logs.

Reply:
I think this will do. Or can we create a role that's not going to confuse people if they look up what resources are being used (if I saw a lambda role being attached to something ECS-related I might question it)?

Reply:
Ok, I have updated to make this way less restrictive as @leezen suggests - I believe we are in a better shape now